src/cljdoc/render/layout.clj

```diff
@@ -78,11 +78,50 @@
    {:style {:top (str TOP-BAR-HEIGHT "px") :left "16rem"}} ; CSS HACK
    contents])
 
+(defn meta-info-icon []
+  [:img#meta-icon.ma3
+   {:src "/meta-info-icon.svg"
```

Comment:
We've been using icons via https://icon.now.sh in some places, which is pretty nice because there's a lot of stuff and we don't have to maintain/select icons on a one by one basis. There's various help icons but also some other interesting ones that might work in this situation like this [explore one](https://icon.now.sh/explore/48/357edd). 

![](https://icon.now.sh/explore/48/357edd) ![](https://icon.now.sh/help_outline/48/357edd) ![](https://icon.now.sh/info_outline/48/357edd)

Reply:
ahh cool, that's super useful to know! I'll exchange the svg for one of those icons 😃